cmd/lifecycle/creator.go

```diff
@@ -276,14 +276,7 @@ func (c *createCmd) Exec() error {
 
 func (c *createCmd) registryImages() []string {
 	var registryImages []string
-	if c.cacheImageRef != "" {
-		registryImages = append(registryImages, c.cacheImageRef)
-	}
-	if !c.useDaemon {
-		registryImages = append(registryImages, append([]string{c.outputImageRef}, c.additionalTags...)...)
-		registryImages = append(registryImages, c.runImageRef, c.previousImageRef)
-	}
-	return registryImages
+	return append(append(registryImages, c.ReadableRegistryImages()...), c.WriteableRegistryImages()...)
```

Comment:
I think it'll be clearer to not use a nested `append`.
I would either split it to the following two lines:
```
append(registryImages, c.ReadableRegistryImages()...)
return append(registryImages, c.WriteableRegistryImages()...)
```
or I would append the output of `c.ReadableRegistryImages()` and `c.WriteableRegistryImages()` and then append them together to `registryImages`.

Reply:
I used two appends in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/7b1d75feefd76dc76fb94799ed2756eeda7da14c)